SurveyApp/Views/AnswerView/TextFieldAnswerView.swift

```diff
@@ -70,10 +74,12 @@ class TextFieldAnswerView: UIView {
         stackView.alignment = .fill
         stackView.distribution = .fillEqually
 
-        for answer in answers {
+        for (index, answer) in answers.enumerated() {
             let textField = CustomTextField()
+            textField.tag = index
             textField.placeholder = answer.inputPlaceholder
             textField.backgroundColor = UIColor.gray.withAlphaComponent(0.8)
+            textField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingDidEnd)
```

Comment:
I think you should use `editingDidChange` here so that the textfield updates more often.

Reply:
I updated the code. Thank you 🙇‍♀️